vipps-psp-api.md

```diff
@@ -173,6 +173,10 @@ Vipps redirects the end user to the `redirectUrl` provided during payment initia
 
 **Please note:**
 
+- The normal
+  [timeouts](https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/timeouts/)
+  apply to the PSP API, and the PSP will get a request to the `makePaymentUrl`
+  after 10 minutes if the user has not acted on the payment request.
```

Comment:
"The PSP will get a request to the `makePaymentUrl`"

request seems like the wrong word ... it will make a request or get a reply...

Reply:
Do you mean: If the user has not acted on the payment request within 10 minutes, the PSP will send a new request to the `makePaymentUrl`?

Reply:
Vipps backend will do a POST towards /makepayment with status=timeout